src/tools/docker-run-to-docker-compose-converter/docker-run-to-docker-compose-converter.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <n-card>
+    <n-form-item label="Your docker run command:">
```

Comment:
To prevent extra gap below the input 
```suggestion
    <n-form-item label="Your docker run command:" :show-feedback="false">
```